staging/src/k8s.io/client-go/plugin/pkg/client/auth/exec/exec.go

```diff
@@ -173,6 +209,10 @@ func (r *roundTripper) RoundTrip(req *http.Request) (*http.Response, error) {
 	if err != nil {
 		return nil, fmt.Errorf("getting token: %v", err)
 	}
+	// The plugin only provided TLS credentials.
```

Comment:
are we not checking for a refresh on 401 responses for client certs?

Reply:
We are now, I didn't think TLS handshake would succeed on bad certs.